server/test/root-route.spec.js

```diff
@@ -5,16 +5,15 @@ import app from '../../index';
 chai.use(chaiHttp);
 const { expect } = chai;
 
-describe('Root URL', () => {
+describe('Root url Test', () => {
   describe('App Root', () => {
-    it('should return status 200 and a message attribute', (done) => {
+    it('should return status 404 and a message attribute', (done) => {
```

Comment:
Bro, well done for the job done so far. But what is the idea behind the `root URL` sending a `404 - Not found` error?